.github/workflows/CodeQL-Analysis.yml

```diff
@@ -0,0 +1,41 @@
+name: "CodeQL"
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 */24 * * 1,3,5'
```

Comment:
This job is scheduled to run according to this Cron Schedule,

<img width="817" alt="Screenshot 2023-03-23 at 6 37 44 PM" src="https://user-images.githubusercontent.com/23498248/227213731-5c93e8b4-eda0-429e-b139-6a1a94080a6a.png">

Reply:
Why would you run this on a schedule? It makes sense on push or PR but I don't see how a schedule would be useful. If you're thinking of vulns, this is already automatically done by github.

Reply:
Hi @NicolasCARPi 
Yes, it makes sense to remove the schedule trigger, I have removed it in the new PR #661